sdcm/cluster.py

```diff
@@ -5380,6 +5399,8 @@ def reconfigure_scylla_monitoring(self):
             for db_node in self.targets["db_cluster"].nodes:
                 monitoring_targets.append(f"{normalize_ipv6_url(getattr(db_node, self.DB_NODES_IP_ADDRESS))}:9180")
             monitoring_targets = " ".join(monitoring_targets)
+            if ssh_connection_ip_type(self.params) != "ipv6":
+                monitoring_targets = monitoring_targets.replace("[", "").replace("]", "")
```

Comment:
When I added this line the code was looks as:
```
            for db_node in self.targets["db_cluster"].nodes:
                monitoring_targets.append(f"[{getattr(db_node, self.DB_NODES_IP_ADDRESS)}]:9180")
```

The brackets were hardcoded added to every address (when it needs in case IPv6 only). Israel changed it at `08/11/2023` and added the brackets for IPv6 address only.
Good catch. Remove it

Reply:
yeah, I've fixed that part already